cmd/genji/main.go

```diff
@@ -26,38 +25,91 @@ func main() {
 		},
 	}
 
+	app.Commands = []*cli.Command{
+		{
+			Name:      "insert",
+			Usage:     "Allows inserting documents from the command line",
+			UsageText: "genji insert [options] [arguments...]",
+			HideHelp:  true,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "engine",
+					Aliases:  []string{"e"},
+					Usage:    "Store engine bolt or badger",
+				},
+				&cli.StringFlag{
+					Name:     "db",
+					Usage:    "path of the database file",
+					Required: true,
+				},
+				&cli.StringFlag{
+					Name:     "table",
+					Aliases:  []string{"t"},
+					Usage:    "name of the table, it must already exist",
+					Required: true,
+				},
+			},
+			Action: func(c *cli.Context) error {
+				dbPath := fmt.Sprintf("%v", c.Value("db"))
```

Comment:
This feels like a hack, the framework provides methods for getting the flags with the right type: https://godoc.org/github.com/urfave/cli#Context.String
```suggestion
				dbPath := c.String("db")
```